tests/manage/rgw/test_rgw_routes.py

```diff
@@ -0,0 +1,113 @@
+import logging
+
+import boto3
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    on_prem_platform_required,
+    skipif_external_mode,
+    tier1,
+)
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.bucket_utils import (
+    s3_delete_object,
+    s3_get_object,
+    s3_put_object,
+)
+from ocs_ci.ocs.exceptions import CommandFailed, UnavailableResourceException
+from ocs_ci.ocs.resources.objectbucket import OBC
+
+log = logging.getLogger(__name__)
+
+
+@on_prem_platform_required
+class TestRGWRoutes:
+    """
+    Test the RGW routes in an ODF cluster
+
+    """
+
+    @skipif_external_mode
+    @bugzilla("2139037")
+    @pytest.mark.polarion_id("OCS-5168")
+    @tier1
+    def test_rgw_routes(self, rgw_bucket_factory):
```

Comment:
please use parametrisation for secure and insecure routes, instead of repeating the code

Reply:
Good catch, done